fenix/app/src/main/java/org/mozilla/fenix/settings/deletebrowsingdata/DeleteBrowsingDataFragment.kt

```diff
@@ -115,10 +115,12 @@ class DeleteBrowsingDataFragment : Fragment(R.layout.fragment_delete_browsing_da
     }
 
     private fun updateDeleteButton(deleteInProgress: Boolean = false) {
-        val enabled = getCheckboxes().any { it.isChecked } && !deleteInProgress
+        context?.let {
```

Comment:
Perhaps consider changing all of this to use `runIfFragmentIsAttached` https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/ext/Fragment.kt#52

Reply:
Done